src/OmniSharp.Host/Internal/OmniSharpConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace OmniSharp.Host.Internal
+{
+    internal static class OmniSharpConstants
+    {
+        internal const string OmnisharpConfigFile = "config.json";
+        internal const string OmnisharpOptionsFile = "omnisharp.json";
+    }
+}
```

Comment:
Multiple references to "OmniSharp" seem redundant. Maybe remove it from the member names? Also, the member names have different casing of "OmniSharp". :smile: